server/src-lib/Hasura/Server/Init/Config.hs

```diff
@@ -109,6 +115,7 @@ data ServeOptions impl
   , soEventsFetchInterval          :: !(Maybe Milliseconds)
   , soLogHeadersFromEnv            :: !Bool
   , soConnectionOptions            :: !WS.ConnectionOptions
+  , soWebsocketKeepAlive           :: !(Maybe KeepAliveDelay)
```

Comment:
Because this value is contained in a `Maybe`, there is duplication of the default value: you have one default value specified in the definition of `webSocketKeepAlive`, and one in the definition of `defaultKeepAliveDelay`. If you make this line into
```suggestion
  , soWebsocketKeepAlive           :: !KeepAliveDelay
```
then this duplication should be avoidable. The type of `rsoWebSocketKeepAlive` can stay `Maybe`'d.